plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/figures/AttributeListFigure.java

```diff
@@ -71,7 +71,7 @@ class ColumnFigureBorder extends AbstractBorder {
 
         @Override
         public Insets getInsets(IFigure figure) {
-            return new Insets(5, 5, 5, 5);
```

Comment:
Why change attributes rendering? We tried this last time and was not good.

Reply:
In class, applied the grid layer that lead to extra inserts is redundant.
Please check on the screen
![image](https://github.com/dbeaver/dbeaver/assets/32031248/3130eba4-c75a-4af9-a1c2-387de7832680)